remotespark/livyclientlib/sparkcontroller.py

```diff
@@ -73,12 +74,17 @@ def add_session(self, name, connection_string, skip_if_exists, properties):
             return
 
         session = self._create_livy_session(connection_string, properties, self.ipython_display)
+
+        spark_events = SparkEvents()
```

Comment:
Why not create one `SparkEvents` object in the constructor and use it for all events on the `SparkController`?